include/fiction/algorithms/simulation/sidb/quickexact.hpp

```diff
@@ -24,6 +24,20 @@
 
 namespace fiction
 {
+/**
+ * Base number required for the correct physical simulation.
+ */
+enum class required_simulation_base_number
+{
+    /**
+     * Three state simulation (i.e., negative, neutral, and positive) is required.
+     */
+    THREE,
+    /**
+     * Two state simulation (i.e., negative and neutral) simulation is sufficient.
+     */
+    TWO
```

Comment:
Is there a special reason for the bases being listed in descending order? :sweat_smile:

Reply:
not really 😅. I will change it